tmt/steps/provision/local.py

```diff
@@ -133,6 +133,34 @@ def pull(
             extend_options: Optional[list[str]] = None) -> None:
         """ Nothing to be done to pull workdir """
 
+    def acquire_log(self, log_name: str) -> Optional[str]:
+        """fetch and return content of a requested log"""
+        if log_name == 'dmesg':
+            log_content = self.execute(Command('dmesg')).stdout
+        if log_content:
+            return log_content
+        return None
+
+    def store_log(self, log_name: str, log_content: str) -> Optional[str]:
+        """save log content and return the path"""
+        if log_name and log_content:
+            self.write(Path(log_name), log_content)
+            return f'{self.workdir}/{log_name}'
+        return None
+
+    def handle_guest_log(self, log_list: Optional[list[str]] = None) -> None:
```

Comment:
Encoding the type in the variable name is usually better to avoid, pure English is often better: `logs`. It's a plural, i.e. "it's a collection" aspect is clearly visible, and it's easy to catch while reading the code.